src/Castle.Facilities.AspNet.SystemWeb.Tests/Castle.Facilities.AspNet.SystemWeb.Tests.csproj

```diff
@@ -19,10 +19,10 @@
 	</PropertyGroup>
 
 	<ItemGroup>
-    <PackageReference Include="NUnit" Version="3.13.3" />
-    <PackageReference Include="NUnit3TestAdapter" Version="4.2.1" />
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.3.2" />
-		<PackageReference Include="Microsoft.TestPlatform.ObjectModel" Version="17.3.2" />
+		<PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.3.2" />
+		<PackageReference Include="Microsoft.TestPlatform.ObjectModel" Version="11.0.0" />
```

Comment:
You've downgrade `Microsoft.TestPlatform.ObjectModel` here which is causing the build to fail.

Reply:
Odd, I remember fighting an issue with this versioning, and needed to downgrade again (had upgraded) to fix it, but can't remember what. Will take a look.

Reply:
There's a couple of versions in use across the solution - `src/Castle.Facilities.AspNet.Mvc.Tests/Castle.Facilities.AspNet.Mvc.Tests.csproj`, `src/Castle.Facilities.AspNet.WebApi.Tests/Castle.Facilities.AspNet.WebApi.Tests.csproj`, and `src/Castle.Windsor.Tests/Castle.Windsor.Tests.csproj` use `11.0.0`, so correction to above, I suspect this one got changed to match. It's _possible_ that it's to work around a test issue with the older NuGet. The problem is alluded to here: https://github.com/microsoft/vstest/issues/1417 - one of the DLLs, I forget which one, has a design-time reference to this old version, never actually used, except Windsor's assembly loading causes it to try. I'll try and remember/work out which one.

Reply:
The change to the package version looks fine. We really need to move to `Directory.Packages.props`.